src/cascadia/TerminalApp/CascadiaSettings.cpp

```diff
@@ -398,6 +398,13 @@ void CascadiaSettings::_CreateDefaultKeybindings()
     keyBindings.SetKeyBinding(ShortcutAction::SwitchToTab8,
                               KeyChord{ KeyModifiers::Alt | KeyModifiers::Ctrl,
                                         static_cast<int>('9') });
+
+    keyBindings.SetKeyBinding(ShortcutAction::IncreaseFontSize,

```

Comment:
My only reservation is that as of #2515, the default keybindings aren't set here anymore. They should instead be included in `src\cascadia\TerminalApp\defaults.json`

Reply:
Interesting conundrum here @zadjii-msft. To adjust text size / zoom using the mouse is <kbd>ctrl</kbd> + scroll. It would therefore, make sense if zoom via the keyboard was also <kbd>ctrl</kbd> + <kbd>+</kbd>/<kbd>-</kbd>.

What would also make THAT even more consistent with mouse scroll actions was if also <kbd>ctrl</kbd> + <kbd>shift</kbd> + <kbd>+</kbd>/<kbd>-</kbd> adjusted blur, but I may be pushing my luck here ... or am I? 😉

Reply:
So, it looks like there's no VT encoding for <kbd>Ctrl-+</kbd>, so it's free to be bound! Problem, though: <kbd>Ctrl+-</kbd> emits `^_`.
We also do need to be considerate of the fact that + is <kbd>Shift+-</kbd>...
Perhaps we should follow suit with other zoomable things and make **<kbd>Ctrl+=</kbd>** the "zoom in" binding?

Reply:
Interesting. <kbd>Ctrl+Shift+-</kbd> emits `^_` in putty _and_ conhost, but <kbd>Ctrl+-</kbd> only emits `^_` in putty. Perhaps the right behavior is that Ctrl alone doesn't do anything and is free to be bound for both?

Reply:
So, like this then?

Key Chord | Keys + Mouse | Action
--- | --- | ---
<kbd>ctrl</kbd> + <kbd>=</kbd>/<kbd>-</kbd> | <kbd>ctrl</kbd> + mouse scroll up/down | Zoom in/out
<kbd>ctrl</kbd>+<kbd>shift</kbd> + <kbd>+</kbd>/<kbd>_</kbd> | <kbd>ctrl</kbd> + <kbd>shift</kbd> + mouse scroll up/down | Blur more/less

Reply:
I've added the bindings to `defaults.json` as
```json
{ "command": "decreaseFontSize", "keys": ["ctrl+-"] },
{ "command": "increaseFontSize", "keys": ["ctrl+="] }
```
If a different default is decided I'll be happy to change it.